src/overlays/actors/ovl_En_Dno/z_en_dno.c

```diff
@@ -40,8 +40,6 @@ void func_80A732C8(EnDno* this, GlobalContext* globalCtx);
 s32 EnDno_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx);
 void EnDno_PostLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx);
 
-extern u8 D_801C20C0;
-
```

Comment:
I see this symbol is defined in undefined_syms.txt too, don't forget to remove it from that file too